processing/src/test/java/org/apache/druid/query/aggregation/any/DoubleAnyVectorAggregatorTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.any;
+
+import org.apache.druid.segment.vector.VectorValueSelector;
+import org.apache.druid.testing.InitializedNullHandlingTest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.mockito.Mockito.spy;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DoubleAnyVectorAggregatorTest extends InitializedNullHandlingTest
+{
+  private static final int NULL_POSITION = 32;
+  private static final int POSITION = 2;
+  private static final double EPSILON = 1e-15;
+  private static final double[] VALUES = new double[]{7.8d, 11, 23.67, 60, 123};
+
+  private ByteBuffer buf;
+  @Mock
+  private VectorValueSelector selector;
+
+  private DoubleAnyVectorAggregator target;
+
+  @Before
+  public void setUp()
+  {
+    byte[] randomBytes = new byte[128];
+    ThreadLocalRandom.current().nextBytes(randomBytes);
+    buf = ByteBuffer.wrap(randomBytes);
+    Mockito.doReturn(VALUES).when(selector).getDoubleVector();
+
+    target = spy(new DoubleAnyVectorAggregator(selector));
+    Mockito.when(target.isValueNull(buf, NULL_POSITION)).thenReturn(true);
+    Mockito.when(target.isValueNull(buf, POSITION)).thenReturn(false);
+  }
+
+  @Test
+  public void initValueShouldInitZero()
+  {
+    target.initValue(buf, POSITION);
+    Assert.assertEquals(0, buf.getDouble(POSITION), EPSILON);
+  }
+
+  @Test
+  public void getAtPositionIsNullShouldReturnNull()
+  {
+    Assert.assertNull(target.get(buf, NULL_POSITION));
+  }
+
+  @Test
+  public void getAtPositionShouldReturnValue()
+  {
+    buf.putDouble(POSITION + 1, VALUES[3]);
```

Comment:
this got me thinking about how did this test pass. There was no flag set in the buffer and the target still returned the value. I realized that in `NumericAnyVectorAggregator`, `non-null` is the default than `null` 
```
boolean isValueNull(ByteBuffer buf, int position)
  {
    return (buf.get(position) & BYTE_FLAG_NULL_MASK) == NullHandling.IS_NULL_BYTE;
  }
```
It should be fine since init is always called. maybe the tests should do the same as well.

Reply:
`Mockito.when(target.isValueNull(buf, POSITION)).thenReturn(false);` should be taking care of it.